### PR TITLE
Update bulk-loading-data-format.rst

### PR DESCRIPTION
**Pull request by lindy65**
Fixed up quotes on json keys

### CODE REVIEW

File: source/deployment/bulk-loading-data-format.rst [hunk 1]
```diff
@@ -79,8 +79,8 @@ For clarity, the object is shown using regular JSON formatting, but in the data
 .. code-block:: javascript
```
Comment (jasonblais) [inline source/deployment/bulk-loading-data-format.rst]: Above this code block, there is one more that doesn't have the proper quotes, below `
The following fragment is from a file that imports two teams, each with two channels, many users, and many posts.`

File: source/deployment/bulk-loading-data-format.rst [hunk 2]
```diff
@@ -79,8 +79,8 @@ For clarity, the object is shown using regular JSON formatting, but in the data
...
 
   {
-    type: "version",
-    version: 1
+    "type": "version",
+    "version": "1"
```
Comment (jasonblais) [inline source/deployment/bulk-loading-data-format.rst]: Actually numbers don't have to be in quotes.

File: source/deployment/bulk-loading-data-format.rst [hunk 3]
```diff
@@ -125,13 +125,13 @@ For clarity, the object is shown using regular JSON formatting, but in the data
...
-    type: "O",
-    description: "The Team Description",
-    allow_open_invite: true
+  "type": "team",
+  "team": {
+    "name": "team-name",
+    "display_name": "Team Display Name",
+    "type": "O",
+    "description": "The Team Description",
+    "allow_open_invite": "true"
```
Comment (jasonblais) [inline source/deployment/bulk-loading-data-format.rst]: @grundleborg Should `true` be in quotes or not?

File: source/deployment/bulk-loading-data-format.rst (hunk 1 above)
Comment (lindy65) [reply source/deployment/bulk-loading-data-format.rst]: Thanks, have fixed :)

File: source/deployment/bulk-loading-data-format.rst (hunk 2 above)
Comment (lindy65) [reply source/deployment/bulk-loading-data-format.rst]: Thanks, have fixed :)

File: source/deployment/bulk-loading-data-format.rst (hunk 3 above)
Comment (grundleborg) [reply source/deployment/bulk-loading-data-format.rst]: None of the values (the stuff to the right of the `:` character) should be changed - they already have quotes if they need them. It's just all the keys that need fixing (the stuff to the left of the `:` character). `true` should *not* be in quotes.

**Pull request merged**